rust/automerge/src/read.rs

```diff
@@ -27,6 +30,11 @@ pub trait ReadDoc {
     /// This function may in future be changed to allow getting the parents from the id of a scalar
     /// value.
     fn parents<O: AsRef<ExId>>(&self, obj: O) -> Result<Parents<'_>, AutomergeError>;
+    fn parents_at<O: AsRef<ExId>>(
```

Comment:
Needs a short docstring explaining that this is the same as `parents` but at a given heads.